public/css/main.css

```diff
@@ -190,72 +190,4 @@ body {
   font-size: 30px;
   z-index: -1;
   right: 5%;
```

Comment:
Please add something like:
```
@media screen and (max-width: 560px) {
   .btn-rollup { display: none } 
}
``` to prevent this on mobile devices: 
![k1](https://user-images.githubusercontent.com/2845077/41965734-91495302-79fd-11e8-807b-d85361090ff6.png)

Ideally using new class (like `.not-important-button`) only for Twitter and Github buttons and leave "How it works" visible.

Reply:
I set 600px because I found that 560px is not enough.